src/services/RetryCountExceededException.js

```diff
@@ -0,0 +1,15 @@
+export default function RetryCountExceededException(requestUnit) {
+  this.message = 'Retry count has been exceeded';
+  this.requestUnit = requestUnit;
+
+  // Use V8's native method if available, otherwise fallback
```

Comment:
http://stackoverflow.com/a/27724419/1008529
`...with the enhancement that stack traces work on Firefox and other browsers. `